crate/kmip/src/crypto/password_derivation.rs

```diff
@@ -10,13 +10,16 @@ use crate::error::KmipError;
 #[cfg(feature = "fips")]
 use crate::kmip_bail;
 
+/// Minimum random salt size in bytes to use when deriving keys.
 const FIPS_MIN_SALT_SIZE: usize = 16;
```

Comment:
`size in bytes` and `size in bits` are a bit confusing. Does it make sense to uniformize them in the same unity ? Could we adopt a naming variable convention to make it clear that those constants are in bytes or in bits. Also, I see `XXX_KLEN`, `XXX_HLEN`, `XXX_SIZE`, `XXX_SIZE_BITS`. Should we use a unique suffix to those variables (_SIZE for example?)